primer/brand
# Updates to Card

> [!Important]  
> PR includes visual breaking changes. 

## Summary

Updates appearance of card to match recommendations from Site Refactor designs. Adds hasShadow and hasBorder properties and removes the shadow and border by default.

Storybook: https://primer-79150425a0-26139705.drafts.github.io/storybook/?path=/story/components-card-features--icon-colors&globals=colorMode:dark

## List of notable changes:


- **added** hasBorder properties
- **updated** border radius of the card container to 16px
- **updated** spacing between body text and link
- **removed** shadow
- **updated** accent color in link
- **updated** stories and tests

## What should reviewers focus on?

- Try it out in [Storybook](https://primer-79150425a0-26139705.drafts.github.io/storybook/?path=/story/components-card-features--icon-colors&globals=colorMode:dark
) and Documentation
- Providing feedback on the new API

## Steps to test:

1. Go to this [Storybook story](https://primer-79150425a0-26139705.drafts.github.io/storybook/?path=/story/components-card-features--icon-colors&globals=colorMode:dark)
2. Check the design matches the proposal in light and dark mode

## Supporting resources (related issues, external links, etc):

- Figma design files: https://www.figma.com/file/X2R5cDZwKGXEGpkjJ4Lf8e/%40josepmartins%2Fcard-updates?type=design&node-id=5407-32782&mode=design&t=m8GYCI3CYfbunpeq-4
- Site design proposal: https://github.com/github/primer/issues/2558
- Tracking issue: https://github.com/github/primer/issues/2609

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

> Please try to provide before and after screenshots or videos

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

<img width="1355" alt="Screenshot 2023-08-31 at 17 11 52" src="https://github.com/primer/brand/assets/912236/60edadfd-cd25-4d27-9c3e-bc01aba97f84">



 </td>
<td valign="top">
<img width="1352" alt="Screenshot 2023-08-31 at 17 10 32" src="https://github.com/primer/brand/assets/912236/19bd0154-9a0d-422c-b242-49d65077a317">




</td>
</tr>
</table>
